Makefile

```diff
@@ -114,14 +114,18 @@ event-handler:
 
 # Run all tests
 .PHONY: test
-test: test-tooling test-terraform
+test: test-tooling test-unit test-terraform
 	@echo Testing plugins against Teleport $(TELEPORT_GET_VERSION)
 	go test -race -count 1 $(shell go list ./... | grep -v -e '/terraform/' -e '/tooling/')
 
 .PHONY: test-tooling
 test-tooling:
 	(cd tooling; go test -v -race ./...)
 
+.PHONY: test-unit
+test-unit:
+	(cd lib; go test -v -race ./...)
```

Comment:
:nail_care: 
We might be testing twice the `lib/`

We can change the `go test` of the `test` target to:
```shell
	go test -race -count 1 $(shell go list ./... | grep -v -e '/terraform/' -e '/tooling/' -e '/lib/')
```